src/core/window/interface.js

```diff
@@ -83,19 +83,21 @@ export class WindowInterface {
 
   /**
    * @static
-   * @param {!Window} win
+   * @param {Window} win
    * @return {typeof XMLHttpRequest}
    */
   static getXMLHttpRequest(win) {
-    return win.XMLHttpRequest;
+    // TODO(wg-performance): This and Image are globals, not part of the Window
```

Comment:
We often grab things off window that are technically global just to aid w/ test stubbing/DI.

Reply:
Artifact from before I understood this (initial pass at this is months old); removed.